app/components/curved-container.tsx

```diff
@@ -35,14 +35,28 @@ export function CurvedContainer({
     ? "drop-shadow-bottom"
     : "hidden";
 
```

Comment:
Adjusting padding for the curved element and adding conditional logic for if the curves themselves appear. Really, the body should probably be abstracted to its own component at this point.